Parser/Client/Browser.php

```diff
@@ -338,7 +339,7 @@ class Browser extends AbstractClientParser
             'CL', 'SU', 'EU', 'UB', 'LO', 'VG', 'TV', 'A0', '1B',
             'S4', 'EE', 'AE', 'VM', 'O0', 'TG', 'GB', 'SY', 'HH',
             'YJ', 'LL', 'TU', 'XV', 'C2', 'QU', 'YN', 'JB', 'MR',
-            'S6', '7S', 'NM', 'PB', 'NR', 'SG',
+            'S6', '7S', 'NM', 'PB', 'NR', 'SG', 'LE',
```

Comment:
I think we can't add this browser as browser family chrome
the browser has 2 engines (IE, Chromium) with the ability to switch
IE Mode:
```
Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; rv:11.0) like Gecko Core/1.63.5847.400 SLBrowser/10.0.2559.400
```
Chrome Mode:
```
Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.25 Safari/537.36 Core/1.70.3706.400 SLBrowser/10.0.3974.400
```